### PR TITLE
Adding Simple external control server example

### PR DESCRIPTION
**Pull request by urrsk**
Adding a python3 server script that can be used to serve the URCap, as mentioned in #21

### CODE REVIEW

**Comment by urrsk** [comment]
Thanks on all the good suggestion, and I agree.
I will look at it when I have time.

File: examples/simple_external_control_server/simple_external_control_server.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# A simple python 3 server to test the External Control URCap
```
[the remark this answers is not in the document]
Comment (urrsk) [reply examples/simple_external_control_server/simple_external_control_server.py:33]: Just updated it to BSD3

[the remark this answers is not in the document]
Comment (fmauch) [reply examples/simple_external_control_server/simple_external_control_server.py:33]: Given that the rest of the repo is released under Apache 2.0 this is rather confusing.

File: examples/simple_external_control_server/simple_external_control_server.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+# The server answer request from the URCap with the context of the given file
+
+import socket
+import socketserver
+import threading
+import argparse
+
+parser = argparse.ArgumentParser(description='Simple External Control server')
+parser.add_argument(
+    "file", type=str, help="Path to the UR script file, which will be send to the robot")
```
Comment (fmauch) [inline examples/simple_external_control_server/simple_external_control_server.py:43]:
```suggestion
    "file", type=str, help="Path to the UR script file, which will be sent to the robot")
```

File: examples/simple_external_control_server/Readme.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Simple External Control Server example
+This simple server, serves the External Control with URScript provided by a file
+
+Example of use:
+```bash
+python3 simple_external_control_server.py -p 50002 hello_world.script
```
Comment (fmauch) [inline examples/simple_external_control_server/Readme.md:6]: We could omit the port parameter in the example execution, as it does have a default value. However, it also does not hurt to specify it, so feel free to ignore this comment.

**Review by fmauch — COMMENTED** [review]
Maybe we should stick with Apache-2.0 for this PR and discuss the licensing issue separately?

**Comment by urrsk** [comment]
@fmauch License changed back as agreed on.

**Review by fmauch — APPROVED** [review]
lgtm

**Pull request merged**